### PR TITLE
docs/technical/work_checklist.md: Add

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by duncandewhurst** [comment]
Rather than adding a new page, I think this content mostly belongs in https://ofds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html with the exception of the detailed instructions for running the tests, which can be added to https://ofds-standard-development-handbook.readthedocs.io/en/latest/technical/index.html.

**Comment by odscjames** [comment]
Seems odd to add to the index page, was that a copy and paste mistake maybe? But this PR does so and then the content will be in the docs, and can be moved around by others later.

File: docs/technical/index.md [hunk 1]
```diff
@@ -7,3 +7,21 @@
 build
 update
 ```
+
+## Running tests locally
+
+When you push code to GitHub tests are automatically run. You will not be allowed to merge your pull request until these tests pass.
```
Comment (lgs85) [inline docs/technical/index.md:13]:
```suggestion
When you push code to GitHub, tests are automatically run. You will not be allowed to merge your pull request until these tests pass. 
```

File: docs/technical/index.md [hunk 2]
```diff
@@ -7,3 +7,21 @@
 build
 update
 ```
+
+## Running tests locally
+
+When you push code to GitHub tests are automatically run. You will not be allowed to merge your pull request until these tests pass. 
+
+To run these locally, [first create and install a virtual environment as instructed on the build page](build).
```
Comment (lgs85) [inline docs/technical/index.md:15]:
```suggestion
To run these locally, first create and install a virtual environment [as instructed on the build page](build).
```

**Review by lgs85 — APPROVED** [review]
Looks good, a couple of v minor copy suggestions

File: docs/technical/index.md (hunk 2 above)
Comment (odscjames) [reply docs/technical/index.md]: Done

File: docs/technical/index.md (hunk 1 above)
Comment (odscjames) [reply docs/technical/index.md]: Done

**Pull request merged**